sirmordred/task_panels.py

```diff
@@ -128,6 +142,20 @@
     ]
 }
 
+JENKINS_EXPORT = "jenkins-export"
+JENKINS_EXPORT_PANEL_OVERALL = "panels/json/jenkins_export.json"
+JENKINS_IP = "panels/json/jenkins-index-pattern.json"
+
+JENKINS_EXPORT_MENU = {
```

Comment:
I don't think this needs a menu. It is already linked in other panel and it's part of Jenkins datasource.

We just need to import this panel in the system.

Reply:
PR updated, please re-check @sduenas